### PR TITLE
LPA-3249 baseline .idea file for phpstorm on refunds

### PR DESCRIPTION
**Pull request by williamfalconeruk**
## Purpose
Brings project run configs into source control for  LPA Refunds 

## Learning
This will break your current PHPStorm config without appropriate preparation. 

 * Exit PHPStorm
 * Backup current run folder  
   `mv .idea .idea-bak`
 * Checkout master/branch
 * Start PHPStorm
 * Remove backup (optional)  
   `rm -R .idea-bak`

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [x] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: .idea/opg-refunds.iml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/public-front/vendor/alphagov/notifications-php-client" />
```
Comment (cooperaj) [inline .idea/opg-refunds.iml]: Do we know why all these excludes exist?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,1296 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cooperaj) [inline .idea/workspace.xml]: This file is ignored but also commited. That shouldn't be the case (it should be ignored)

**Comment by cooperaj** [comment]
Not actually seeing any run configs in here?

File: .idea/workspace.xml (hunk 2 above)
Comment (williamfalconeruk) [reply .idea/workspace.xml]: hmm - will delete that locally - let me take another look.

**Comment by williamfalconeruk** [comment]
> Not actually seeing any run configs in here?

Odd - i'll take another stab at this..

File: .idea/opg-refunds.iml (hunk 1 above)
Comment (williamfalconeruk) [reply .idea/opg-refunds.iml]: not a clue - i can remove this if needed.

Comment (cooperaj) [reply .idea/opg-refunds.iml]: Yeah. I think if you look at the PR that went into Use these were removed at somepoint.

**Comment by cooperaj** [comment]
Acutally, could be worth looking at the naming of the various run configs. It's a little redundant to suffix them as "test" as for the most part unless they're compound configs they are presented in the dropdown lists as tests

**Comment by williamfalconeruk** [comment]
@cooperaj - ready for rereview

**Pull request merged**